### PR TITLE
Publish module method, and pub/sub example.

### PR DESCRIPTION
**Pull request by digitalextremist**
This not only adds support to cover #334, it also brings in an example for the wiki so we don't need to only reference `Reel`

### CODE REVIEW

**Comment by digitalextremist** [comment]
@tarcieri for future reference, as with here -- is it more efficient to create a method which redirects to another method as with 2d7a982, or is it better to `extend Forwardable` and delegate one method. I always think extending `Forwardable` for one method isn't worth the trouble if it's going to impact so many other things downstream in something as large as `Celluloid`.

File: lib/celluloid/notifications.rb [hunk 1]
```diff
@@ -80,4 +81,8 @@ def matches?(subscriber_or_pattern)
       end
     end
   end
+
+  def self.publish *args
```
Comment (tarcieri) [inline lib/celluloid/notifications.rb]: Parens?

I guess we should get Rubocop hooked up for real...

**Comment by tarcieri** [comment]
This sort of thunk is fine if it's just a method

File: lib/celluloid/notifications.rb (hunk 1 above)
Comment (digitalextremist) [inline lib/celluloid/notifications.rb]: fixed in 15d5cfe

**Pull request merged**